### ISSUE TITLE
Handle symphony delete files

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/searchworks_traject_indexer#61]

**Comment by mejackreed** [comment]
@shelleydoljack how often do these deletes occur?

**Comment by shelleydoljack** [comment]
All the time.

**Comment by shelleydoljack** [comment]
The delete file is generated as part of the hourly. I think there is also a delete file for the nightly run too.

**Comment by mejackreed** [comment]
Gotcha.. so these are the files like this?

```
180718_ckeys_delete.del
180718_uni_increment.marc
```

Those seem to be tied to a date, I guess the hourlies are somewhere else?

**Comment by shelleydoljack** [comment]
The hourly files are like this:
```
-rw-r--r-- 1 sirsi sirsi     4962 Aug  9 09:01 ckeys_delete.del
-rw-r--r-- 1 sirsi sirsi 14073459 Aug  9 09:02 uni_partday.marc
-rw-r--r-- 1 sirsi sirsi      201 Aug  9 09:02 dates_counts
```
See the script https://github.com/sul-dlss/solrmarc-sw/blob/master/stanford-sw/scripts/pullThenIndexBodoniHourly.sh

**Comment by mejackreed** [comment]
Resolved in #65
